release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1099,6 +1099,8 @@ This script removes unauthenticated subjects from the following cluster role bin
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1821771[*BZ#1821771*])
 
+* If you use a persistent volume (PV) for Prometheus and are running monitoring on the minimum recommended nodes, which is 2 core and 8 GB, you might experience OOM kills during upgrade to {product-title} 4.7. To avoid this issue when upgrading your cluster, allow nodes with double the size of memory at 16 GB. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925061[*BZ#1925061*])
```

Comment:
I think it's good that we call out clusters running with the minimal supported footprint but we should also mention that in general, the cluster should have enough free capacity to support Prometheus doubling its memory usage during the upgrade.
@s-urbaniak wdyt?

Reply:
Yes agreed, we have seen OOM kills in other larger clusters. Maybe just doubling memory during upgrades to prevent any problems?

Reply:
sounds good making this more general :+1: